src/org/exist/storage/BrokerPool.java

```diff
@@ -1004,8 +758,9 @@ protected void initialize() throws EXistException, DatabaseConfigurationExceptio
 
                             sync(broker, Sync.MAJOR);
 
+                            //TODO(AR) remove this!
                             //require to allow access by BrokerPool.getInstance();
-                            instances.put(instanceName, this);
+                            //instances.put(instanceName, this);
```

Comment:
without that it's not possible to get `BrokerPool` by `BrokerPool.getInstance()`